keyboards/converter/ps2_usb/rules.mk

```diff
@@ -0,0 +1,23 @@
+MCU = atmega32u4
+F_CPU ?= 16000000
+ARCH ?= AVR8
+F_USB ?= $(F_CPU)
+OPT_DEFS += -DSUSPEND_MODE_STANDBY
+BOOTLOADER_SIZE = 4096
+
+BOOTMAGIC_ENABLE ?= no
+MOUSEKEY_ENABLE ?= yes
+EXTRAKEY_ENABLE ?= yes
+CONSOLE_ENABLE ?= yes
+COMMAND_ENABLE ?= yes
+NKRO_ENABLE ?= yes
+
+#PS2_USE_USART ?= no
+PS2_USE_INT ?= yes
+#PS2_USE_BUSYWAIT ?= no
+
+CUSTOM_MATRIX ?= yes
+
+SRC = matrix.c led.c
+
+LAYOUTS = 104_ansi 105_iso ibm_f_at
```

Comment:
This rules is only for the [Layouts feature](https://docs.qmk.fm/#/feature_layouts). Your converter could be made to use it, but it currently doesn't, so for now this line should be removed.

```suggestion
```